viewing_party/party.py

```diff
@@ -0,0 +1,199 @@
+    if len(watchedlist)==0:
+      return None
+    else:
+      most_frequent=max(count_list)
+      index_of_most_frequent=count_list.index(most_frequent)
+      max_genre=genre_list[index_of_most_frequent]
+      return max_genre
+
+def get_unique_watched(user_data):
+  #user_data={'watched':[{},{}],'friends':[{'watched':[{'title':,'genre':,'rating':},{}]},{}]
+
+  user_watched=user_data['watched']
+  #which will return a list of dictionaries of my films
+  #i will then loop through to get all titles
+
+  user_watched_titles=[]
+  for i in range(len(user_watched)):
+      user_watched_title=user_watched[i]
+      user_watched_titles.append(user_watched_title)
+
+  friends_watched_titles=[]
+  friends_watched=user_data['friends']
+  for j in range(len(friends_watched)):
+    friend_ind_watched=friends_watched[j]['watched']
+    for k in range(len(friend_ind_watched)):
+      friends_watched_titles.append(friend_ind_watched[k])
+  unique_watched=[]
+  for title in user_watched_titles:
+    if title not in friends_watched_titles:
+      unique_watched.append(title)
+  
+  return unique_watched
+
+def get_friends_unique_watched(user_data):
+  
+  user_watched=user_data['watched']
+  #which will return a list of dictionaries of my films
+  #i will then loop through to get all titles
+
+  user_watched_titles=[]
+  for i in range(len(user_watched)):
+      user_watched_title=user_watched[i]
+      user_watched_titles.append(user_watched_title)
+
+  friends_watched_titles=[]
+  friends_watched=user_data['friends']
+  for j in range(len(friends_watched)):
+    friend_ind_watched=friends_watched[j]['watched']
+    for k in range(len(friend_ind_watched)):
+      friends_watched_titles.append(friend_ind_watched[k])
+
+  unique_watched=[]
+  for title in friends_watched_titles:
+    if title not in user_watched_titles:
+      if title not in unique_watched:
+        unique_watched.append(title)
+
+  return unique_watched
+
+def get_available_recs(user_data):
+
+  recommended_movies=[]
+  friends_movies=[]
+  hosts=[]
+  friends_final_list=[]
+  watched_final_list=[]
+  for i in range(len(user_data['friends'])):
+    friends_watched=user_data['friends'][i]['watched']
+    for j in range(len(friends_watched)):
+        friends_watched_host=friends_watched[j]['host']
+        hosts.append(friends_watched_host)
+        friends_watched_title=friends_watched[j]['title']
+        friends_movies.append(friends_watched_title)
+        friends_final_list.append(friends_watched[j])
+  for p in range(len(user_data['watched'])):
+    watched=user_data['watched'][p]['title']
+    watched_final_list.append(watched)
+
+  for movie in friends_final_list:
+   index=friends_final_list.index(movie)
+   host=hosts[index]
+   if host in user_data['subscriptions']: 
+     if not movie['title'] in watched_final_list and not movie in recommended_movies:
+        recommended_movies.append(movie)
+  return recommended_movies
+
+def get_new_rec_by_genre(user_data):
```

Comment:
think about how you can use `get_most_watched_genre(user_data), get_friends_unique_watched(user_data), and get_unique_watched(user_data)`